Backend/Matchmaking/src/main/java/matchmaking/manager/MatchupManager.java

```diff
@@ -33,8 +33,9 @@ public class MatchupManager {
 
     // A placeholder for "no player" in case of odd number of participants
     private final Signups NO_PLAYER = new Signups()
-            .setUuid("null")
-            .setTournamentId(null)
+            .setId(new PlayerTournamentId()
+                    .setUuid("null")
```

Comment:
is this to do with the database column not allowing null?

Reply:
changed naming to be consistent with RatingUtil